TaskManager/Modules/Add Button/View/AddTaskViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  AddTaskViewController.swift
+//  TaskManager
+//
+//  Created by user on 22.09.2022.
+//
+
+import UIKit
+
+protocol AddTaskView: AnyObject {
+    func titleAndDescription(title: String, description: String?)
+}
+
+class AddTaskViewController: UIViewController, AddTaskView {
+    
+    @IBOutlet weak private var addTaskLabel: UILabel!
+    @IBOutlet weak var titleTextField: UITextField!
+    @IBOutlet weak var subtitleTextField: UITextField!
+    @IBOutlet weak var createTaskButton: UIButton!
+    
+    var delegate: HomeView?
+    var presenter: AddViewPresenter!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setBarItem()
+        titleTextField.addTarget(self, action: #selector(hiddenButton), for: .editingChanged)
+        setTextField()
+    }
+    
+    func titleAndDescription(title: String, description: String?) {
+        titleTextField.text = title
+        subtitleTextField.text = description
+    }
+    
+    private func setTextField() {
+        titleTextField.delegate = self
+        titleTextField.layer.cornerRadius = 8
```

Comment:
Usually we create a struct in class and have static properties with constants. In this way we can find them easily if we need to change them and also use them in multiple locations